### PR TITLE
introduce in master page syntax

### PR DESCRIPTION
**Pull request by vicancy**

### CODE REVIEW

**Comment by superyyrrzz** [comment]
:shipit:

**Comment by hellosnow** [comment]
:shipit:

File: test/Microsoft.DocAsCode.Build.Engine.Tests/TemplateManagerUnitTest.cs [hunk 1]
```diff
@@ -573,6 +653,80 @@ public void TestLiquidTemplateProcessSingleTemplateWithDependenciesShouldWork()
...
+            string partial2 = @"partial 2:
+{% for item in model -%}
+{{ item.name }}
+{% endfor -%}
+";
+
+            string master = @"
+Hello Master
+{% include partial1 -%}
+{%- body %}
```
Comment (hellosnow) [inline test/Microsoft.DocAsCode.Build.Engine.Tests/TemplateManagerUnitTest.cs]: minor: update pair to {%- x -%} or {% x %}? starts with %- and ends without -% is not align?

Comment (vicancy) [inline test/Microsoft.DocAsCode.Build.Engine.Tests/TemplateManagerUnitTest.cs]: This is to test `%` with `-` also works

**Pull request merged**